harmoni_actuators/harmoni_web/web/src/config/config.json

```diff
@@ -1,13 +1,54 @@
 {
```

Comment:
We might want to add an issue for to make a generator for these configs. It looks like it would be a bit of a pain to do by hand.

Reply:
This should be an issue, not part of this request